wled00/um_manager.cpp

```
@@ -18,7 +18,12 @@ static size_t getCount() {
 
 
 //Usermod Manager internals
-void UsermodManager::setup()             { for (auto mod = DYNARRAY_BEGIN(usermods); mod < DYNARRAY_END(usermods); ++mod) (*mod)->setup(); }
+void UsermodManager::setup() {
+  for (auto mod = DYNARRAY_BEGIN(usermods); mod < DYNARRAY_END(usermods); ++mod) {
```

Cppcheck (2.21.0)

[error] Comparing pointers that point to different objects

(comparePointers)